ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp

```diff
@@ -685,6 +726,29 @@ void py_module(py::module& module) {
         ttnn::remainder,
         R"doc(Perform an eltwise-modulus operation a - a.div(b, rounding_mode=floor) * b.", "Support provided only for WH_B0.)doc");
 
+    detail::bind_inplace_operation(
+        module,
+        ttnn::gt_,
+        R"doc(Perform Greater than in-place operation on :attr:`input_tensor` and :attr:`other` and returns the tensor with the same layout as :attr:`input_tensor`
```

Comment:
change :attr: name for all four ops here to match whatever py::arg name you decide to use

Reply:
sure @KalaivaniMCW. Will change it.